lib/sdf-server/src/service/diagram/create_connection.rs

```diff
@@ -112,6 +113,28 @@ pub async fn create_connection(
                 .await?;
         }
     }
+    let to_component_name = to_component.name(&ctx).await?;
+    let to_socket_name = InputSocket::get_by_id(&ctx, request.to_socket_id)
+        .await?
+        .name()
+        .to_string();
+    ctx.write_audit_log(
+        AuditLogKind::CreateConnection {
+            from_component_id: request.from_component_id,
+            from_component_name: from_component.name(&ctx).await?,
+            from_socket_id: request.from_socket_id,
+            from_socket_name: OutputSocket::get_by_id(&ctx, request.from_socket_id)
+                .await?
+                .name()
+                .to_string(),
+            to_component_id: request.to_component_id,
+            to_component_name: to_component_name.clone(),
+            to_socket_id: request.to_socket_id,
+            to_socket_name: to_socket_name.clone(),
+        },
+        format!("{0}-{1}", to_component_name, to_socket_name),
```

Comment:
nit for the future: I'm a bit surprised the linter didn't catch this, but you can do the following:
```rust
format!("{to_component_name}-{to_socket_name}"),
```

Reply:
You know, I didn't realize you could use positional numbers like `0` and `1`. Before `format!` was able to use `{variable}` style syntax, the best you could do was `format!("{}-{}", to_component_name, to_socket_name)`. Anyway, @nickgerace's idea is the most current Rustic way, agreed!

Reply:
@fnichol year the positional numbers are interesting! I've used them in the past, moreso when I want to display something multiple times. They're cool!

Reply:
I will update this too!